### ISSUE TITLE
Validate NC geometry with a paper-faithful ResNet9/MNIST positive control

### ISSUE THREAD

**Issue by contra333** [contra333/2026-_0707#148]
## Goal

Add and validate one bounded positive-control path that reproduces the paper's ResNet9/MNIST fixed-total Adam coupling experiment with the project's optimizer and NC geometry implementations, then run the lightest scientifically interpretable server experiment.

## Research reason

This tests whether the project pipeline can recover the reference paper's within-architecture coupled-decay geometry trend under a close architecture and recipe. It is a validation control, not a new main experiment and not evidence that any optimizer universally produces more collapse.

## Read first

- `AGENTS.md`
- `docs/PROJECT_CONTEXT.md`
- `docs/WORKFLOW.md`
- `docs/STATUS.md`
- `docs/reference_cards/05_training_protocol.md`
- `docs/reference_cards/09_core_representation_metrics.md`
- `docs/reference_cards/10_optimizer_grid_literature_anchors.md`
- `docs/reference_cards/11_metric_contract_v1_2.md`
- `docs/reference_cards/13_active_paper_protocol.md`
- `docs/sources.lock.yaml`

## Allowed scope

- A validation-only ResNet9 model matching `jydzhao/neural_collapse_optimizer@7cab4a59bc28da6e356cee1e793ec67a694933b9`.
- A validation-only MNIST loader matching the pinned transform recipe.
- A small runner/config matrix using project Adam coupled-decoupled semantics and project NC0-NC4 geometry code.
- Focused model, data, optimizer/config, geometry, provenance, and smoke tests.
- A separate protocol addendum and compact result report for this positive control.
- Read-only server resource inspection and, after execution identity and resource bounds are frozen, a bounded GPU execution.

## Forbidden changes

- No modification or reinterpretation of frozen WRN-28-10, ResNet-18, Task F, protected OOD, or existing result artifacts.
- No FashionMNIST/OOD evaluation.
- No rescue hyperparameter grid, CIFAR-100, checkpoint inference outside this new validation namespace, or large artifact download.
- Do not silently repair the upstream entrypoint or describe reconstructed code as the authors' exact runnable program.

## Acceptance criteria

- [ ] ResNet9 structure, input channels, feature dimension, classifier orientation, and logits reconstruction match the pinned upstream source.
- [ ] MNIST resize/normalization and no-augmentation contract match the pinned upstream source.
- [ ] Six arms hold total decay at `5e-4` while coupled ratio is `{0,.2,.4,.6,.8,1}`; ratio orientation and all-parameter decay scope are explicit.
- [ ] Adam endpoints and mixed updates pass independent parity tests under the project's standard pre-step decoupled semantics; the upstream post-step discrepancy is recorded.
- [ ] NC0, raw NC2-ETF, raw NC3, accuracy, and supporting NC1/NC4 are evaluated from deterministic train/test loaders using the project geometry implementation.
- [ ] Same seed gives sibling arms identical initialization and first-epoch data order.
- [ ] A bounded server smoke run succeeds before the frozen execution.
- [ ] The result report separates exact recipe checks, directional geometry comparison, and non-comparable absolute values.

### Local or desktop

```bash
python -m compileall src scripts tests
pytest -q <focused ResNet9 positive-control tests>
```

### Department server

```bash
pytest -q <focused ResNet9 positive-control tests>
python <positive-control runner> --max-epochs 1 <single-arm smoke arguments>
python <positive-control runner> <frozen six-arm arguments>
```

## Expected outputs

- Validation-only code, configs, focused tests, and protocol/source provenance.
- Hash-addressed server run artifacts outside the Git checkout.
- Compact JSON/Markdown comparison containing per-arm trajectories and endpoint metrics.

## Not included

- Re-running existing WSL suites solely to reproduce old environment evidence.
- Multi-architecture absolute NC comparison, causal mediation, OOD evaluation, or modification of the paper's main claims.

## Unverified assumptions

- One seed may establish a pipeline positive control but cannot establish multi-seed stability. A second/third seed is conditional on runtime and first-seed signal quality.
- The paper's printed NC2/NC3 scaling conflicts with its code/table magnitudes, so raw upstream-compatible values are primary for directional comparison.
- The upstream repository's committed main entrypoint is broken; only pinned architecture, config, transforms, optimizer intent, and raw metric conventions are treated as reconstructable evidence.

**Comment by contra333** [comment]
Owner confirmed that the ResNet9/MNIST positive-control work is no longer needed. PR #149 was closed and its local/remote branches were removed during the approved branch cleanup.
